civihr_default_theme/scss/base/_variables.scss

```diff
@@ -163,3 +163,8 @@ $crm-scrollbar-thumb-color: $crm-gray-light-blue !default;
 $crm-customselect-height:        33 !default;
 $crm-customselect-caret-width:   32 !default;
 $crm-customselect-padding-right: $crm-customselect-caret-width + 12 !default;
+
+//== Forms
+//
+//##
+$input-border:              $crm-color-neutral !default;
```

Comment:
I have checked that `$input-border`is not used anywhere else other than the `form-control`, also `$crm-color-neutral` was used previously for defining the border which was wrong. So using `input-border` makes sense as it internally is same as `$crm-color-neutral`.